programs/vote/src/vote_processor.rs

```diff
@@ -1798,11 +1798,12 @@ mod tests {
     #[test]
```

Comment:
odd that no test exists that would have caught this bug 🤔

Reply:
I will add one.